extension/textext/asktext.py

```diff
@@ -716,6 +719,11 @@ def cb_key_press(self, widget, event, data=None):
                 self._ok_button.clicked()
                 return True
 
+            # Show/ update Preview shortcut (CTRL+P)
+            if Gdk.keyval_name(event.keyval) == 'p' and Gdk.ModifierType.CONTROL_MASK and event.state:
+                self._preview_button.clicked()
```

Comment:
Shortcut needs to be disabled for the time preview is being generated, and reactivated again on success/compilation error as well as preview button

Reply:
Will provide separate PR for this. Same holds for save button. Also, if they are clicked by the mouse.

Reply:
Ok